core/common/src/main/java/alluxio/QuietlyCloseable.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import java.io.Closeable;
+
+/**
+ * A Closeable which doesn't throw any checked exceptions from its {@link #close()} method.
+ */
+public interface QuietlyCloseable extends Closeable {
```

Comment:
FYI, instead of introducing `QuietlyClosable` you can change redeclared the method again in an interface that extends `Closable` with a more strict signature